src/test/resources/conditionalizePermissions/silicon_0008.vpr

```diff
@@ -0,0 +1,15 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+
+field f: Int
+
+method deAlias(x: Ref, y: Ref, k: Perm)
+  requires k >= none
+  requires acc(x.f, k) && acc(y.f, k)
+  // The following postcondition should fail in default greedy Silicon because --conditionalizePermissions creates a
+  // situation where the two chunks may or may not alias in the same branch, so they cannot definitively be merged,
+  // so greedy Silicon cannot prove the postcondition using any individual chunk.
+  //:: ExpectedOutput(postcondition.violated:insufficient.permission)
+  ensures x == y ==> acc(x.f, 2 * k)
```

Comment:
doesn't this postcondition fail regardless of the algorithm because we do not constrain `k` to be at most `1/2`?

Reply:
I think I don't understand what you mean. The postcondition promises 2k permissions, right, so no matter what k is (and there isn't anything that would require it at least 1/2 as far as I can see), if x == y, that should hold and we have 2k permission (but Silicon greedy can't prove it after the rewriting).

Reply:
Ah, nevermind, Somehow I failed to realize that we were using a `k` in the perm amount of `x.f` and `y.f`. Please ignore this comment